static/usr/lib/systemd/system/console-conf@.service

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=Ubuntu Core Firstboot Configuration %I
+After=systemd-user-sessions.service plymouth-quit-wait.service
+After=rc-local.service
+# on core18/core20 the firstboot seeding happens via the core[18].start-snapd service,
+# make sure to start after so that the 'snap' command is available
+After=core18.start-snapd.service core.start-snapd.service
+# on core20 the user may invoke a recovery chooser, make sure the detection
+# service runs before
+After=snapd.recovery-chooser-trigger.service
+IgnoreOnIsolate=yes
+ConditionPathExists=/dev/tty0
+ConditionPathExists=|!/var/lib/console-conf/complete
+ConditionPathExists=|/run/snapd-recovery-chooser-triggered
+StartLimitInterval=0
```

Comment:
This needs an explanation I think. Original issue: https://bugs.launchpad.net/snappy/+bug/1650091

Reply:
Can you add the link to the issue in a comment? So we can keep track later if we can remove by testing the case of the issue.

Reply:
@bboozzoo

Reply:
this is done now, I've left a comment with explanation and a link to the bug report